_notices/rsn0009.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 9 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Drop Ubuntu 16.04 in v0.20"
```

Comment:
Maybe `Stop supporting Ubuntu 16.04 in v0.20 / v21.06.0`?
Stop supporting sounds friendlier, and we can mention the new calver

Reply:
Agreed